config/routes.rb

```diff
@@ -340,6 +340,12 @@
         get 'get_marking_scheme_details'
         get 'download_csv_grades_report'
       end
+
+      member do
+        get 'view_summary'
+        get 'index'
```

Comment:
Remove this route. There's already a default `index` route for collection that you should use if you really need to, but this is probably unused.